src/libmongoc/tests/unified/operation.c

```diff
@@ -2761,6 +2761,49 @@ operation_assert_number_connections_checked_out (test_t *test,
    return true;
 }
 
+static bool
+operation_rename (test_t *test,
+                  operation_t *op,
+                  result_t *result,
+                  bson_error_t *error)
+{
+   // First validate the arguments
+   const char *object = op->object;
+   bson_iter_t it;
+   bool okay = bson_iter_init_find (&it, op->arguments, "to");
+   if (!okay) {
+      test_set_error (error, "'rename' operation is missing a 'to' argument");
+      return false;
+   }
+   if (!BSON_ITER_HOLDS_UTF8 (&it)) {
+      test_set_error (error, "'rename' argument is not a utf8 string");
+      return false;
+   }
+
+   // Now get the entity
+   entity_t *ent = entity_map_get (test->entity_map, object, error);
```

Comment:
Variable declarations may need to be hoisted to satisfy Visual Studio 2013 builds.

Reply:
VS2013 supports intermixed variable declarations

Reply:
Oh, nice. TIL.